src/scripts/emailBreachAlerts.js

```diff
@@ -187,11 +193,34 @@ export async function poll(subClient, receivedMessages) {
                 utmCampaign: utmCampaignId,
               };
 
+              // try to append a new row into the email notifications table
+              // if the append fails, there might be already an entry, stop the script
+              try {
+                await addEmailNotification({
+                  breachId,
+                  subscriberId,
+                  notified: false,
+                  email: data.recipientEmail,
+                  notificationType: "incident"
+                })
+              } catch(e) {
+                console.error("Failed to add email notification to table: ", e)
+                throw new Error(e)
+              }
+
               const emailTemplate = getTemplate(data, breachAlertEmailPartial);
               const subject = getMessage("breach-alert-subject");
 
               await sendEmail(data.recipientEmail, subject, emailTemplate);
 
+              // mark email as notified in database
+              // if this call ever fails, stop stop the script with an error
+              try{
```

Comment:
I'm confused... It looks like Prettier is linting these files, but not sure why it wouldn't format a space after the `try` and before `{` to match L198 above.

Reply:
We are actually ignoring the `src/scripts` folder for some reason. Good catch, turned that on